packages/models-library/src/models_library/settings/services_common.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseSettings, PositiveInt
+
+
+class ServicesCommonSettings(BaseSettings):
```

Comment:
I am ok  that ``models_library/settings`` could be a place but I would like to suggest a change of how you are "packaging" the settings to avoid ending up with a flat list of very complicated names permutations :-)

I would suggest to follow the same strategy as docker-compose does to specify the [``deploy`` constraints](https://docs.docker.com/compose/compose-file/#deploy) (e.g. policies, configs, etc).  

First, we could use hierarchies to distinguish e.g. the service's client it refers. 
Then, each policy/config can be in a separate entry so they can be reused for each service.

This is how it would evolve (there is not need to implement all now):
```python

class RetryPolicy(BaseModel):
   wait: Optional[PositiveInt] = None

   ... 
   # embed some of the  tenacity configs
   # define some with defaults for all clients??

class EntryPoint(BaseModel):
     host:  ...
     port:  ...
   
#-- client-focus common settings ---------

class StorageClientSettings(BaseSettings):
     retry_policy: Optional[RetryPolicies] = None      
     request_timeout: PositiveInt = Field(60*60, description="SOMETHING INFORMATIVE")

    class Config:
        env_prefix = "STORAGE_CLIENT_"

class DirectorClientSettings(BaseSettings):
     retry_policy: Optional[RetryPolicies] = None
     stop_request_timeout: PositiveInt ...

     class Config:
        env_prefix = "DIRECTOR_CLIENT_"
    
```

Moreover, this way we could easily integrate all the *retry policies* for redis and [postgres](https://github.com/ITISFoundation/osparc-simcore/blob/d0acfaf96dc729b2bca5b7056c9b24893f941b74/packages/service-library/src/servicelib/aiopg_utils.py) that we have already in place.

When we move to py3.7 we could even use generic models https://pydantic-docs.helpmanual.io/usage/models/#generic-models